include/odp/api/spec/ipsec.h

```diff
@@ -262,6 +263,14 @@ typedef struct odp_ipsec_capability_t {
 	/** Supported authentication algorithms */
 	odp_crypto_auth_algos_t   auths;
 
+	/** Support of traffic manager (TM) after inline IPSEC processing. On
+	 *  unsupported platforms, inline IPSEC outbound operations would be
```

Comment:
This second sentence could be just removed as we do not need to promise this sort of behavior. The two basics behavior we can choose from for the case the capability is not there, are: 1) Application is not allowed to use a pktio with TM enabled with inline IPsec. 2) Application is allowed to use such pktio with inline IPsec but the IPsec packets do not go through TM. Petri an I think that the first one is the better choice. My other comments are based on that.

Reply:
I'll reword to, 

"On unsupported platforms, application is not allowed to use a TM enabled pktio with inline IPSEC."

Can you confirm if this is fine?